ios/freighter-mobile/Info.plist

```diff
@@ -44,6 +44,8 @@
 	</dict>
 	<key>NSCameraUsageDescription</key>
 	<string>Freighter needs access to your Camera to scan QR codes.</string>
+	<key>NSFaceIDUsageDescription</key>
+	<string>Freighter needs access to your Face ID to unlock the app.</string>
```

Comment:
In addition to this, we need to add the `en.lproj` and `pt.lproj` strings on XCode for `NSFaceIDUsageDescription` so the pop-up gets translated like we have for `NSUserTrackingUsageDescription`. Not related to this PR, but would you mind also adding the translations for `NSCameraUsageDescription` which are also missing? Thanks!